UI/Shared/GameGrid.razor

```diff
@@ -0,0 +1,32 @@
+@using TicTacGo.Enums
+
+<div class="bg-indigo-600 grid grid-cols-3 grid-rows-3 gap-2 h-80 w-80 m-auto mt-2 mb-2 p-2 rounded-lg">
+    @for (int i = 0; i < GridCells.Length; i++)
+    {
+        Marks cell = GridCells[i];
+        int Index = i;
```

Comment:
Not necessary. `i` is already an `int` and `i` is standard in all programming languages for the index within a for loop.